commit 0a0cb1de0fac12aaa087ca9b1f50f202271b3560 (HEAD)
Author: Seth Junot <xsetech@gmail.com>
Date:   Mon Feb 16 10:59:07 2026 -0800

    Fix missing <cstdlib> import to resolve malloc and free

diff --git a/include/fmt/format.h b/include/fmt/format.h
index 50e57144..4d2719b3 100644
--- a/include/fmt/format.h
+++ b/include/fmt/format.h
@@ -41,6 +41,7 @@
 #include "base.h"
 
 #ifndef FMT_MODULE
+#  include <cstdlib>  // malloc and free
 #  include <cmath>    // std::signbit
 #  include <cstddef>  // std::byte
 #  include <cstdint>  // uint32_t
